api/controllers/UserController.js

```diff
@@ -2014,15 +2384,30 @@ module.exports = {
       user,
     };
     await Promise.all([
-      user.save(),
-      NotificationService.send(notification),
+      user.save().then(() => {
```

Comment:
Same as above, the `connection_request` email may be sent even if the `user.save()` fails.

Reply:
geez. well hey the nice thing is the code will be easier to read after using plain async/await

Reply:
HID-2102